url/url.go

```diff
@@ -52,3 +54,24 @@ func mergeQuery(originalQuery url.Values, newQuery url.Values) url.Values {
 func stripSlashes(str string) string {
 	return strings.Trim(str, "/")
 }
+
+// Attempt to open a URL in the user's browser. We use this to open docs, PRs we've
+// programmatically opened, etc
+func openURL(url string) error {
```

Comment:
Make this public (`OpenURL`)

Reply:
Ah yeah that would probably help! :face_exhaling:

Reply:
Done in ba66eab